doc/classes/SpatialMaterial.xml

```diff
@@ -157,10 +157,17 @@
 			Specifies whether to use [code]UV[/code] or [code]UV2[/code] for the detail layer. See [enum DetailUV] for options.
 		</member>
 		<member name="distance_fade_max_distance" type="float" setter="set_distance_fade_max_distance" getter="get_distance_fade_max_distance">
-			Distance at which the object fades fully and is no longer visible.
+            Distance at which the object appears at normal opaqueness.
+            
+            [b]Note:[/b] If max_distance is less than min_distance, then the behavior will be defined as:
```

Comment:
change to "[b]Note:[/b] If [code]distance_fade_max_distance[/code] is less than [code]distance_fade_min_distance[/code], the behavior will be reversed, the object will start to fade away at [code]distance_fade_max_distance[/code] and fully disappear once it reaches [code]distance_fade_min_distance[/code].